roles/openshift_master/tasks/main.yml

```diff
@@ -114,12 +121,26 @@
 
 - name: Start and enable openshift-master
   service: name=openshift-master enabled=yes state=started
+  when: not openshift_master_ha
   register: start_result
 
 - name: pause to prevent service restart from interfering with bootstrapping
   pause: seconds=30
   when: start_result | changed
 
+- name: Install cluster packagese
```

Comment:
Hey, it's like I invented a new language...  Could be @maxamillion's native language...  packagese